### ISSUE TITLE
bug: Waiting for Socket message (UDP)

### ISSUE THREAD

**Issue by djoice** [PassTheMayo/minecraft-server-util#38]
Hi! Thx for library! I saw a small problem: some servers do not return a response (socket message), as a result, the connection remains and no response / error is received. Example addresses:

```
87.98.244.30:25570
play.fireflype.tk:19132
mc.crazynetwork.xyz:19132
45.13.59.162:25744
```

This error occurs on all methods where UDP Socket is used. (status, statusBedrock)
Perhaps we need to add a response timeout or an error so that we can correctly process the server response.

**Comment by djoice** [comment]
Thx!

**Comment by PassTheMayo** [comment]
I’ll try to look into this issue as it’s still a problem in the latest release. I’ll use this to track progress.

**Comment by PassTheMayo** [comment]
I have just published `3.6.0` which should hopefully resolve this issue. Please update to this version and verify the bug is resolved.
